partiql-planner/src/main/kotlin/org/partiql/planner/internal/transforms/RelConverter.kt

```diff
@@ -923,36 +927,17 @@ internal object RelConverter {
     }
 
     /**
-     * Rewrites a SELECT node replacing (and extracting) each aggregation `i` with a synthetic field name `$agg_i`.
```

Comment:
We shouldn't be deleting the content of the prior comment related to the synthetic field names.